aws_datalake/README.md

```diff
@@ -88,22 +88,25 @@ resource "aws_s3_bucket" "segment_datalake_s3" {
 # Creates the IAM Policy that allows Segment to access the necessary resources
 # in your AWS account for loading your data.
 module "iam" {
-  source = "git@github.com:segmentio/terraform-segment-data-lake//aws_datalake/modules/iam?ref=v0.8.0"
+  source = "git@github.com:segmentio/terraform-segment-data-lakes//aws_datalake/modules/iam?ref=v0.8.0"
 
   # Suffix is not strictly required if only initializing this module once.
   # However, if you need to initialize multiple times across different Terraform
   # workspaces, this hook allows the generated IAM policies to be given unique
   # names.
   suffix = "-prod"
-
+  
+  # segment_region is an optional field which helps you to specify the region for your datalake. The default is us_west_2
```

Comment:
I would rename this to "...specify the region if you have configured Segment in a different region. See https://segment.com/docs/guides/regional-segment/ for more details". Or something on those lines to make it clear.